example/tutorial/04_simple_memoryspaces/simple_memoryspaces.cpp

```diff
@@ -61,12 +61,12 @@ typedef Kokkos::View<double * [3]> view_type;
 // performance penalties then it is its own host_mirror_space. This is
 // the case for HostSpace, CudaUVMSpace and CudaHostPinnedSpace.
 
-typedef view_type::HostMirror host_view_type;
+using host_view_type = view_type::HostMirror;
```

Comment:
I'm kind of surprised this doesn't require `typename`?

Reply:
@masterleinad did you check whether that file gets compiled at all?

Reply:
Yes, that file gets compiled and the compiler doesn't complain with the new version.